README.md

```diff
@@ -2,46 +2,43 @@
 
```

Comment:
All of these changes to the README.md are from my `Prettier` extension; can be ignored.

Reply:
It's really not a great idea to include changes like this in an unrelated PR.  It makes code review and merging much harder.  We can't just "ignore" them, because in this case, there was a typo introduced in another branch that this would persist.

And it's likely to create merge conflicts.  I want to encourage you to restore this file to the way it is on the main branch, and if you want to update the formatting, do it in a separate PR.

Reply:
I've reverted the README back to the main branch! Everything should be cleared now :)

Also, yes I agree. I just wanted to use my individual pitest coverage stuff on this branch xD